README.md

```diff
@@ -87,9 +87,9 @@ We refer to any programming language you can write quantum circuits in as a *fro
-|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| <a href="https://quantumai.google/cirq"><img src="https://raw.githubusercontent.com/quantumlib/Cirq/master/docs/images/Cirq_logo_color.png" alt="Cirq logo" width="130"/></a> | <a href="https://qiskit.org/"><img src="https://qiskit.org/images/qiskit-logo.png" alt="Qiskit logo" width="80"/></a> | <a href="https://github.com/rigetti/pyquil"><img src="https://www.rigetti.com/uploads/Logos/logo-rigetti-gray.jpg" alt="Rigetti logo" width="150"/></a> | <a href="https://github.com/aws/amazon-braket-sdk-python"><img src="https://a0.awsstatic.com/libra-css/images/logos/aws_logo_smile_1200x630.png" alt="AWS logo" width="150"/></a> |    &nbsp;&nbsp;  <a href="https://pennylane.ai/"><img src="https://pennylane.ai/img/xanadu_x.png" alt="PennyLane logo" width="60"/></a> |
+| [Cirq](https://quantumai.google/cirq) 0.14.0                                                                                                                                  | [Qiskit](https://qiskit.org/) 0.35.0                                                                                   | [pyQuil](https://github.com/rigetti/pyquil) 3.0.0                                                                                                       | [Braket](https://github.com/aws/amazon-braket-sdk-python) 1.17.0                                                                                                                  | [PennyLane](https://pennylane.ai/) 0.22.2                                                                                               |
```

Comment:
Not sure about hard-coding versions. But I am ok with it if you think it's important.

Reply:
It was part of the original issue suggestion, happy to pull as yeah hard-coding is a bad code smell :)

Reply:
Rolled back